test/e2e/storage/persistent_volumes-local.go

```diff
@@ -254,47 +263,32 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		var testVol *localTestVolume
 
 		BeforeEach(func() {
-			testVol = setupLocalVolumePVCPV(config, node0)
+			testVol = setupLocalVolumePVCPV(config, DefaultMountPointType)
 		})
 
 		AfterEach(func() {
 			cleanupLocalVolume(config, testVol)
 		})
 
 		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2", func() {
-			By("Creating pod1 to write to the PV")
-			pod1, pod1Err := createLocalPod(config, testVol)
-			Expect(pod1Err).NotTo(HaveOccurred())
-
-			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod1))
-			pod1NodeName, pod1NodeNameErr := podNodeName(config, pod1)
-			Expect(pod1NodeNameErr).NotTo(HaveOccurred())
-			framework.Logf("Pod1 %q created on Node %q", pod1.Name, pod1NodeName)
-			Expect(pod1NodeName).To(Equal(node0.Name))
-
-			By("Creating pod2 to read from the PV")
-			pod2, pod2Err := createLocalPod(config, testVol)
-			Expect(pod2Err).NotTo(HaveOccurred())
+			mountPointTypeIt(config, testVol)
+		})
+	})
 
-			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod2))
-			pod2NodeName, pod2NodeNameErr := podNodeName(config, pod2)
-			Expect(pod2NodeNameErr).NotTo(HaveOccurred())
-			framework.Logf("Pod2 %q created on Node %q", pod2.Name, pod2NodeName)
-			Expect(pod2NodeName).To(Equal(node0.Name))
+	Context("with true tmpfs mountpoint when two pods request one prebound PVC at the same time", func() {
 
-			writeCmd, readCmd := createWriteAndReadCmds(volumeDir, testFile, testVol.hostDir /*writeTestFileContent*/)
+		var testVol *localTestVolume
 
-			By("Writing in pod1")
-			podRWCmdExec(pod1, writeCmd)
-			By("Reading in pod2")
-			readOut := podRWCmdExec(pod2, readCmd)
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, TmpfsMountPointType)
+		})
 
-			Expect(readOut).To(ContainSubstring(testVol.hostDir)) /*aka writeTestFileContents*/
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+		})
 
-			By("Deleting pod1")
-			framework.DeletePodOrFail(config.client, config.ns, pod1.Name)
-			By("Deleting pod2")
-			framework.DeletePodOrFail(config.client, config.ns, pod2.Name)
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2", func() {
```

Comment:
Would it also be useful to add the test case of two pods serially accessing the volume?